iterative_verifier.go

```diff
@@ -572,6 +573,27 @@ func (v *IterativeVerifier) tableIsIgnored(table *schema.Table) bool {
 	return false
 }
 
+func (v *IterativeVerifier) verifiedColumns(table *schema.Table) []schema.TableColumn {
```

Comment:
nitpick: The name makes it sound like the columns are already verified. How about `columnsToVerify`?